build.gradle

```diff
@@ -700,6 +711,14 @@ project('spring-integration-xml') {
 
 		testCompile "xmlunit:xmlunit:$xmlUnitVersion"
 	}
+
+	if (JavaVersion.current() != JavaVersion.VERSION_1_8) {
+		[compileTestJava]*.options*.compilerArgs = ['--add-modules', 'java.xml.bind']
```

Comment:
You are missing in all these manipulation our global `xLintArg` option.
I wonder if we should move all the `--add-modules` to the top level config, along side with our `[compileJava, compileTestJava]*.options*.compilerArgs = [xLintArg]` there...

Reply:
Perhaps `+=` works here?

Reply:
No, I tried :cry: